packages/odyssey-babel-plugin/src/normalizeOpts.ts

```diff
@@ -26,7 +26,7 @@ export function normalizeOpts(
 ): NormalizedOpts {
   const opts = babelOpts || Object.create(null);
   const normalized = {
-    include: [/\.module\.scss$/i],
+    include: [/\.module\.(?:scss|css)$/i],
```

Comment:
No issues with this, just curious if we ran into somewhere `.css` inclusion was necessary.

Reply:
Yes! I used this approach for my demo app as I didn't want to bring in sass too!